.circleci/choose-e2e-env.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/bash
 set -xv
 
+# PR branch only
 if [[ ! -z $CIRCLE_PULL_REQUEST ]]; then
-  # Count total number of files changed
-  CHANGED_COUNT=$(git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) | wc -l | xargs)
-
-  # Count total number of e2e files
-  E2E_CHANGED_COUNT=$(git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) -- e2e | wc -l | xargs)
-
-  # If e2e files are the only files that have changed, update WORKBENCH_ENV to "test"
-  if [ $E2E_CHANGED_COUNT -gt 0 ] && [ $E2E_CHANGED_COUNT -eq $CHANGED_COUNT ]; then
+  # Count total number of changed files that are not e2e and circleci
```

Comment:
The new comments and logic are hard to follow, I think there are too many negatives. I've read it about 3-4 times and I'm still not entirely sure this is correct

Should the logic just be: if changes in ui/ directory (or yaml) -> run against local env, else run against test - why the focus on the test directories?